### PR TITLE
handle comment stripping via transition table.

### PR DESCRIPTION
**Pull request by maditya**
Comment stripping will be made configurable in the subsequent PR

@yukinying @adon-at-work @neraliu please review!
This is to address https://github.com/yahoo/html-purify/issues/10#issuecomment-126760856

### CODE REVIEW

**Comment by yukinying** [comment]
+1

**Comment by yahoocla** [comment]
CLA is valid!

File: src/derived-states.js [hunk 1]
```diff
@@ -24,13 +23,6 @@ DerivedState.TransitionsSparse = {
...
     42: {1: 2, 43: 6},
     43: {1: 2},
-    44: {1: 1, 44: 1},
-    45: {44: 1, 53: 1},
-    46: {48: 1},
-    48: {48: 1, 49: 1},
-    49: {48: 1, 50: 1},
-    50: {1: 1},
-    53: {46: 1}
 };
```
Comment (adon-at-work) [inline src/derived-states.js:26]: `TransitionsSparse` is not used in practice, but apparently designed only for generating the `Transitions` table.
Consider commenting it out? may be to reduce the size for `dist/`

**Comment by adon-at-work** [comment]
Without considering configurability, this method is much cleaner, and I can also see the performance improvement.

good to merge. :+1:

File: src/derived-states.js (hunk 1 above)
Comment (maditya) [inline src/derived-states.js:26]: sure, will do in the next PR.

**Pull request merged**